flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -89,5 +90,32 @@ message DeleteTaskRequest {
 }
 
 // Response to delete a task.
-message DeleteTaskResponse {
+message DeleteTaskResponse {}
+
+// A message containing the agent metadata.
+message Agent {
```

Comment:
Not adding `async` here?

Reply:
This is for both `sync` and `async` Agent, do you think we need to add it?